### PR TITLE
feat: test.each support string template

### PR DESCRIPTION
**Pull request by poyoho**
fix: #2335

### CODE REVIEW

**Review by sheremet-va — CHANGES_REQUESTED** [review]
Thank you for the PR, but it needs more work:

1. Vitest also has `describe.each`, and it should support the same API
2. We need documentation for this feature

**Comment by poyoho** [comment]
ok. wait me later

File: docs/api/index.md [hunk 1]
```diff
@@ -243,6 +243,23 @@ You cannot use this syntax, when using Vitest as [type checker](/guide/testing-t
   // ✓ add(2, 1) -> 3
   ```
 
+  You can also access template string table.
+    * First row of variable name column headings separated with `|`
+    * One or more subsequent rows of data supplied as template literal expressions using ${value} syntax.
+
+    ```ts
+    test.each`
+    a    | b    | expected
```
Comment (sheremet-va) [inline docs/api/index.md:252]: Let's make it more readable, can you add more spaces, so `|` character visually forms a table?

File: docs/api/index.md [hunk 2]
```diff
@@ -571,6 +588,25 @@ You cannot use this syntax, when using Vitest as [type checker](/guide/testing-t
...
 
+  You can also access template string table.
+    * First row of variable name column headings separated with `|`
+    * One or more subsequent rows of data supplied as template literal expressions using ${value} syntax.
+
+    ```ts
+    describe.each`
+    a    | b    | expected
+    ${1} | ${1} | ${2}
+    ${'a'} | ${'b'} | ${'ab'}
```
Comment (sheremet-va) [inline docs/api/index.md:599]: The same here

**Review by sheremet-va — CHANGES_REQUESTED** [review]

**Review by sheremet-va — APPROVED** [review]

**Pull request merged**